src/SwipeableViews.spec.js

```diff
@@ -29,6 +29,77 @@ describe('SwipeableViews', () => {
     });
   });
 
+  describe('prop: hysteresis', () => {
```

Comment:
I should start adding some e2e tests infrastructure. That would make a great test.